packages/openapi-ts/src/plugins/@pinia-colada/sdk/config.ts

```diff
@@ -0,0 +1,21 @@
+import type { Plugin } from '../../types';
+import { handler } from './plugin';
+import type { Config } from './types';
+
+export const defaultConfig: Plugin.Config<Config> = {
+  _dependencies: ['@hey-api/typescript'],
+  _handler: handler,
+  exportFromIndex: true,
+  name: '@pinia-colada/sdk',
```

Comment:
Please keep the name consistent with the underlying library, i.e. `@pinia/colada`. Unless you had a different reason for naming it this way?

Reply:
I figured it was going to take more than one plugin to cover different implementations of `@pinia/colada` and `Pinia` itself wouldn't really have anything to do with this library. There's also Vue Data-Loaders that can be integrated for endpoints on top of the Pinia Colada SDK, but would require a much more complex interface, but still massively benefit as an `openapi-ts` plugin. Maybe a different directory structure would make sense, but the dependent relationship between implementations/plugins would be nice have communicated with the directory structure.

Reply:
Can you link me to the docs or examples for data loaders integration with Pinia Colada? My impression is you'd still be installing a single package (`@pinia/colada`) so it makes sense to serve it with a single plugin

Reply:
https://uvr.esm.is/data-loaders/
https://uvr.esm.is/data-loaders/colada/
I think the initial implementation was part of Pinia Colada, but it's dependent on plugging into the Vue component mounting through the router, so it looks like it's more agnostic now and "can" be used without Pinia Colada but I think that's still where the first-class support is.

Reply:
Yeah so this is a separate plugin/workflow. I could be convinced there to have a separate plugin for that approach, but I stand behind making the core plugin match `@pinia/colada` in naming, that makes it easier for everyone to discover and use!